__tests__/selenium_tests/e2e_tests.ts

```diff
@@ -198,6 +362,9 @@ describe('End to end test', () => {
 
       logger.info(`found url contains deployment`)
       await driver.wait(until.urlContains('/deployment/'))
+      await driver.sleep(2000)
```

Comment:
We already wait until the deployment contains `/deployment/`, why are we waiting more?

Reply:
This was causing issues, but I can take it out and try again. Sometimes the tests don't wait until the components fully render, and I'm not sure how else to do it.

Reply:
Seems to be working without it :)